src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-size: cover;
```

Comment:
remove background-size and add background-repeat: no-repeat;
  background-position: center;